var/spack/repos/builtin/packages/py-hostlist/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHostlist(PythonPackage):
```

Comment:
Package should be named `py-python-hostlist` since [python-hostlist](https://pypi.org/project/python-hostlist/) and [hostlist](https://pypi.org/project/hostlist/) are two different things

Reply:
I am sorry about that, I got confused with the [`py-hostlist`](https://github.com/LLNL/py-hostlist) repo.. Thanks for the correction! I fixed the git url too.